### PR TITLE
[WIP] Update for swift 5

### PR DESCRIPTION
**Pull request by barrault01**
Hello I run the xcode migration, I run the tests and they are passing.

### CODE REVIEW

**Comment by daniel-beard** [comment]
Any chance of someone merging this one soon and getting a new cocoapods release?

**Comment by pauloec** [comment]
Unlikely. The owner has left indefinitely.

**Comment by iosdevben** [comment]
Thanks for doing the PR @barrault01. I've switched my Cartfile to point to your repo instead.

File: .github/FUNDING.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# These are supported funding model platforms
```
Comment (JoeMatt) [inline .github/FUNDING.yml]: Can we remove this from the PR and open another with it if it's wanted

**Comment by JoeMatt** [comment]
`.swift-version` should be updated to 5.0 also.

**Comment by kuyazee** [comment]
> `.swift-version` should be updated to 5.0 also.

Shouldn’t we remove the `.swift-version` file because of the specification in the Podspec?

**Comment by JoeMatt** [comment]
> > `.swift-version` should be updated to 5.0 also.
> 
> Shouldn’t we remove the `.swift-version` file because of the specification in the Podspec?

I agree, .swift-version is considered deprecated by Cocoapods. Other tools may use it but it was never a standard for swift. Package.swift or the XCode project already reflect their compatible version.s

**Comment by kuyazee** [comment]
> > > `.swift-version` should be updated to 5.0 also.
> > 
> > 
> > Shouldn’t we remove the `.swift-version` file because of the specification in the Podspec?
> 
> I agree, .swift-version is considered deprecated by Cocoapods. Other tools may use it but it was never a standard for swift. Package.swift or the XCode project already reflect their compatible version.s

Ok then, once the `.swift-version` file is removed we can merge this PR. Thanks guys

**Comment by barrault01** [comment]
Guys going to close this pull request I just run the xcode migration, and now with Xcode 11 it is not building the tests anymore... I will let my repository available until someone migrate the lib to xcode 11 + Swift 5.

**Comment by JoeMatt** [comment]
Reopening so we can build on this work

**Comment by jdisho** [comment]
What is the status of this PR? Is somebody working on?

**Comment by JoeMatt** [comment]
> What is the status of this PR? Is somebody working on?
> If not, I would prefer taking it over.

You're welcome to copy this branch and make a new PR and reference this issue number #594. Looks like it just needs a rebase and conflict resolution.

**Comment by JoeMatt** [comment]
duplicated by #650 #695

**Pull request closed without merging**